proto/zetachain/zetacore/crosschain/cross_chain_tx.proto

```diff
@@ -65,6 +65,7 @@ message OutboundParams {
   uint64 tss_nonce = 5;
   uint64 gas_limit = 6;
   string gas_price = 7;
+  string gas_priority_fee = 13;
```

Comment:
Has `13` already been used before at some point? Maybe `23` is for the next new field ? @lumtis

Reply:
I don't remember why we jumped on the number like this. I don't think we ever used `13`

Reply:
Okay, I'll bump it. 

To prevent this in the future, we can integrate `buf breaking` into CI (https://buf.build/docs/breaking/overview). It works great, however sometimes it makes life harder rather than easier 😅  cc @gartnera

Reply:
https://github.com/zeta-chain/node/pull/2483/commits/fa69f4873705dd2418563a58a49253c9c609b1c5